src/index.js

```diff
@@ -1,18 +1,19 @@
-import React from 'react';
+/* eslint-disable import/no-named-as-default */
```

Comment:
- [OPTIONAL] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `eslint`, it will be nice to fix the issue.